system_tests/snap_sync_test.go

```diff
@@ -0,0 +1,171 @@
+// Copyright 2021-2022, Offchain Labs, Inc.
+// For license information, see https://github.com/nitro/blob/master/LICENSE
+
+package arbtest
+
+import (
+	"context"
+	"math/big"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/ethereum/go-ethereum/params"
+
+	"github.com/offchainlabs/nitro/arbnode"
+	"github.com/offchainlabs/nitro/arbos/l2pricing"
+	"github.com/offchainlabs/nitro/util"
+)
+
+func TestSnapSync(t *testing.T) {
+	ctx, cancelCtx := context.WithCancel(context.Background())
+	defer cancelCtx()
+
+	var transferGas = util.NormalizeL2GasForL1GasInitial(800_000, params.GWei) // include room for aggregator L1 costs
+
+	// 1st node with sequencer, stays up all the time.
+	builder := NewNodeBuilder(ctx).DefaultConfig(t, true)
+	builder.L2Info = NewBlockChainTestInfo(
+		t,
+		types.NewArbitrumSigner(types.NewLondonSigner(builder.chainConfig.ChainID)), big.NewInt(l2pricing.InitialBaseFeeWei*2),
+		transferGas,
+	)
+	cleanup := builder.Build(t)
+	defer cleanup()
+
+	// 2nd node without sequencer, syncs up to the first node.
+	// This node will be stopped in middle and arbitrumdata will be deleted.
+	nodeB, cleanupB := builder.Build2ndNode(t, &SecondNodeParams{})
```

Comment:
not an absolute must, but I think cleaner:
* create a testdir and use it for stackconfig
* then use the same stackconfig for both nodeB and nodeC
* see TestWasmRecreate

Reply:
Done.